packages/cardano-services/src/Asset/types.ts

```diff
@@ -1,10 +1,21 @@
 import { Asset, Cardano } from '@cardano-sdk/core';
 import { Shutdown } from '@cardano-sdk/util';
 
+export enum MetadataSourceType {
+  Registry = 'registry',
+  OnChain = 'on-chain',
```

Comment:
Given we've not yet implemented `on-chain` or `referenced` support, let's just have a single option in this enum. Also, we'll probably be looking to align with a type registry that [this CIP](https://github.com/cardano-foundation/CIPs/pull/298) introduces.

Reply:
Updated in 050fc2bd1a1b3204fee85152928253535131d24c